infra/scripts/k8s-common-functions.sh

```diff
@@ -30,36 +43,34 @@ function helm_install {
     #   $RELEASE is helm release name
     #   $DOCKER_REPOSITORY is the docker repo containing feast images tagged with $GIT_TAG
     #   ... you can pass additional args to this function that are passed on to helm install
+    #   $NAMESPACE is the namespace name
 
     local RELEASE=$1
     local DOCKER_REPOSITORY=$2
     local GIT_TAG=$3
+    local NAMESPACE=$4
 
-    shift 3
-
-    # Wait for images to be available in the docker repository; ci is the last image built
-    timeout 15m bash -c "while ! gcloud container images list-tags ${DOCKER_REPOSITORY}/feast-ci --format=json | jq -e \".[] | select(.tags[] | contains (\\\"${GIT_TAG}\\\"))\" > /dev/null; do sleep 10s; done"
+    shift 4
 
     # We skip statsd exporter and other metrics stuff since we're not using it anyway, and it
     # has some issues with unbound PVCs (that cause kubectl delete pvc to hang).
-    echo "${STEP_BREADCRUMB} Helm installing feast"
+    echo "${STEP_BREADCRUMB:-} Helm installing feast"
 
-    if ! time helm install --wait "$RELEASE" infra/charts/feast \
-        --timeout 15m \
+    if ! time helm install --wait "$RELEASE" ./infra/charts/feast \
+        --timeout 5m \
```

Comment:
the azure test requires >5min (15 to be safe), I believe because of some services requiring others to be up + crashloopbackoff. Probably could be fixed but I had just increased the timeout

Reply:
Ok rolled this back, but this is peculiar since on EKS it usually takes maybe a minute, and you'd think this is pure k8s stuff that shouldn't depend on the cloud provider much.